bimmer_connected/cli.py

```diff
@@ -6,6 +6,8 @@
 import json
 import os
 import time
+import urllib.request
```

Comment:
I'd suggest to use `requests` for http calls to the server.

Reply:
Agreed, `requests` is used in other parts of the library as well.

Reply:
I can change my code to use requests, if that is preferred. Probably be a couple of days till I can make the changes and test again.

Reply:
I found some time this evening and changed to use requests. I tested and send a few POIs to my car successfully, but good if others can test. 

I git pushed my changes and submitted a new PR. Apologies in advance if I did not follow the right process. Please let me know.